source/configure/calls-deployment.rst

```diff
@@ -327,9 +328,24 @@ If you wish to host many calls or calls with a large number of participants, tak
   # Allow to allocate more memory as needed for more control messages that need to be sent for each socket connected
   net.core.optmem_max = 16777216
 
+Configuring recording
```

Comment:
@streamer45, I'm adding this as a link to ensure that any updates done to that repo/page remain the single source of truth.

Reply:
Yep, that makes sense to me, thanks :+1: